src/ProjectNeon/Assets/Data/Templates/Scope.cs

```diff
@@ -0,0 +1,7 @@
+﻿
+public enum Scope
```

Comment:
This doesn't belong in `Data/Templates`. Should live with the feature folder. `Template` are only for `ScriptableObject`

Reply:
@SilasReinagel Fixed